client/utils/account-fees.tsx

```diff
@@ -124,12 +124,12 @@ export const formatMethodFeesTooltip = (
 
 	const total = {
 		percentage_rate:
-			accountFees.base.percentage_rate +
-			accountFees.additional.percentage_rate +
+			accountFees.base.percentage_rate * discountAdjustedFeeRate +
+			accountFees.additional.percentage_rate * discountAdjustedFeeRate +
 			accountFees.fx.percentage_rate,
 		fixed_rate:
-			accountFees.base.fixed_rate +
-			accountFees.additional.fixed_rate +
+			accountFees.base.fixed_rate * discountAdjustedFeeRate +
+			accountFees.additional.fixed_rate * discountAdjustedFeeRate +
```

Comment:
I did not follow all conversations around the topic, but it looks like discount are being applied to both base/intl. and NOT FX fees. Is that correct?

Reply:
Per our[ terms and conditions ](https://woo.com/es/terms-conditions/woopayments-promotion-2023/)we define payment processing fees as:

> [Payment Processing Fees](https://woo.com/document/woocommerce-payments/fees-and-debits/fees/) include payment transaction fees, card transaction fees (for example, 2.9% plus $0.30 for debit or credit card transactions in the United States), subscription-based fees (for example, the 1% additional fee for a renewal payment processed by WooCommerce Payments’ built-in subscription functionality), and fixed per transaction fees (including international processing charges). Payment Processing Fees do not include fees charged for currency conversion fees, dispute or chargebacks, or instant deposit fees.

Reply:
I see; thanks for sharing the context. Marking the thread as resolved.